bumble/gatt.py

```diff
@@ -256,10 +257,17 @@ def properties_as_string(properties):
             if properties & p
         ])
 
-    def __init__(self, uuid, properties, permissions, value = b'', descriptors = []):
+    @staticmethod
+    def string_to_properties(properties_str: str):
+        return functools.reduce(lambda x,y: x|get_dict_key_by_value(Characteristic.PROPERTY_NAMES, y), properties_str.split(','), 0)
```

Comment:
style: break into multiple lines and spaces after `,` and around `|`  (I suggest `flake8` to detect code style warnings, even though we're not 100% strict about that, it helps) :
```
return functools.reduce(
    lambda x, y: x | get_dict_key_by_value(Characteristic.PROPERTY_NAMES, y),
    properties_str.split(','),
    0
)
```

Reply:
@barbibulle do you think I should verify `get_dict_key_by_value` returns non-None?

Reply:
I think it's best to let this throw if you pass a property name that's not supported.